pkg/convert/go_test.go

```diff
@@ -0,0 +1,67 @@
+package convert
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+var modelMsg = model.Messages{
+	Language: language.English,
+	Messages: []model.Message{
+		{
+			ID:          "1",
+			Message:     "message1",
+			Description: "description1",
+			Fuzzy:       true,
+		},
+		{
+			ID:          "2",
+			Message:     "message2",
+			Description: "description2",
+			Fuzzy:       false,
+		},
+	},
+}
+
+func TestToGo(t *testing.T) {
+	t.Parallel()
+
+	expected := []byte(`{"language":"en","messages":
+[{"id":"1","meaning":"description1","message":"","translation":"message1","fuzzy":true},
+{"id":"2","meaning":"description2","message":"","translation":"message2"}]}`)
+
+	buffer := new(bytes.Buffer)
+	if err := json.Compact(buffer, expected); err != nil {
+		fmt.Println(err)
```

Comment:
You most probably want to assert error, because otherwise when the test will fail, there will be no straight answer on why it failed, due `ToGo` or due  `json.Compact`

Reply:
Same goes for TestFromGo()

Reply:
done ;)